### DISCUSSION TITLE
Change default file name

### DISCUSSION THREAD

**Discussion by cindyyoyo** [MacielG1/VodRecovery#11] [Ideas]
Hello there,

First I'ld like to thank you to take up on that project, adding selenium into it was a nice idea to bypass cloudflare protection that would most of the times block the requests/grequests.. Nice feature also to be able to specify a twitch link directly (haven't tried it yet tho)

My suggestion idea is pretty simple, not the most brilliant but I believe it could benefit many people. I think the downloaded file could be named CHANNEL_DATE.xxx by default instead of CHANNEL_VODID.xxx. Where date format would be YYYY-MM-DD. So the vods directly classify in order in the folder we download them. Maybe add an option in the settings to allow user to specify a naming format with different parameters would be nice.

I'm collecting all vods from a specific channel and I'ld find it very useful if I did not have to rename it manually (although its not really a hassle doing so tbf).

I previously had the occasion to dive into Shishkebaboo's source code so I have some idea of the working flow, but I guess you changed it a bit. However I could do a pull request eventually if you want to delegate this task (not sure when cause I'm involved in other projects tho). 

Thanks for the time you put in, I wish you all the best <3

**Comment by MacielG1** [comment]
Thank you for your feedback! 
I appreciate your suggestion. In the next update, I'll be changing the filename structure for better organization:


from: 
```
streamer_videoId.extension
```

to: 
```
streamer - stream_date - [videoId].extension
```

and for Twitch direct URLs containing the title:

from:
```
title - streamer [videoId].extension
```
to:
```
streamer - stream_date - title - [videoId].extension
```

and for files with specified start and end times, it'll also change accordingly.
